docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+volumes:
+ # static volume will be mounted to both nginx and django gunicorn services.
+ static_volume:
+ # mysql data volume will be mounted to mysql services
+ data_volume:
+
+services:
+  db:
+    build: ./docker/mysql/
+    volumes:
+      - ./data_volume:/var/lib/mysql
+    restart: always
+    env_file:
+      - ./docker/mysql/db-environment.env
+    ports:
+      - "3306:3306"
+  web:
+    build: .
+    command: /code/gunicorn.sh
+#    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - static_volume:/code/static
+    expose:
+      - "8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+  nginx:
+    build: ./docker/nginx/
+    ports:
+      - "8000:8000"
+    volumes:
+      - static_volume:/code/static
+    links:

```

Comment:
Nginx service should be run only after 'merida' service, please fix it.
Do you actually need 'links' in nginx service?